### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by gitondezhuts** [infiniteremote/installer#2] [General]
Please join this discussion if you would like to help translate our API

**Comment by fragli** [comment]
I join

**Reply by gitondezhuts** [reply]
what languages can you translate to?

**Reply by fragli** [reply]
Polish

**Comment by Pofig1sT161** [comment]
I can help. Russian language.

**Comment by mediosdigitalesudem** [comment]
I can help Spanish language

**Comment by k3-cat** [comment]
I can help Chinese (CHS & CHT) language

**Reply by gitondezhuts** [reply]
It is originally in chinese, I translated it to english using chatgpt to start with, then made other changes where I saw fit. https://github.com/kingmo888/rustdesk-api-server Its moved quite alot away from the original as some insecure parts were removed and made it ready to run in production

**Comment by TBG-FR** [comment]
I can help with French language 🙋‍♂️

**Comment by VenimK** [comment]
Can Help with Dutch

**Comment by ziovec** [comment]
I'll be more than happy to help with Italian translation

**Comment by TobiasKWF** [comment]
Can Help with German

**Comment by gitondezhuts** [comment]
can you join discord and I will make a group there where we can discuss this

**Comment by ShogunMan** [comment]
I can do Norwegian.
